web/app.fluidity.money/app/routes/$network/dashboard/airdrop/index.tsx

```diff
@@ -905,7 +888,9 @@ const MyMultiplier = ({
                   }}
                 >
                   <Text prominent code>
-                    {numberToMonetaryString(getUsdFromTokenAmount(amount, 6))}{" "}
+                    {numberToMonetaryString(
+                      2 * getUsdFromTokenAmount(amount, 6)
```

Comment:
This should be memoised somewhere

Reply:
no point - parent page does not rerender often